packages/cli/default/agent.template.yml

```diff
@@ -389,10 +389,12 @@ credentialIssuerLD:
     - suites:
         # - $require: '@veramo/credential-ld#VeramoEd25519Signature2018'
         # - $require: '@veramo/credential-ld#VeramoEd25519Signature2020'
-        - $require: '@veramo/credential-ld#VeramoJsonWebSignature2020'
+        # - $require: '@veramo/credential-ld#VeramoJsonWebSignature2020'
         # - $require: '@veramo/credential-ld#VeramoEcdsaSecp256k1RecoverySignature2020'
+        - $require: '@vckit/credential-data-integrity#VCkitEddsaRdfc2022'
```

Comment:
How can we support V1 and V2 VCDM? is it backward-compatible?

Reply:
Yes, it can be backward-compatible with both V1 and V2 VCDM.